lib/travis/build/script/node_js.rb

```diff
@@ -204,7 +204,6 @@ def install_yarn
               sh.if yarn_req_not_met do
                 sh.echo "Node.js version $(node --version) does not meet requirement for yarn." \
                   " Please use Node.js #{YARN_REQUIRED_NODE_VERSION} or later.", ansi: :red
-                npm_install config[:npm_args]
```

Comment:
This is a more salient question to ask. Is desirable to run `npm install` when the repository clearly indicates the use of `yarn` but the version of Node.js does not meet the requirement.

The current assumes "yes". I chose this behavior when `yarn` was relatively new to the scene and falling back to `npm` made sense.

A couple of years later, this assumption may no longer holds. However, dropping this could break existing builds that rely on this behavior.

Reply:
Hmm. That's a good point. To think, this does seem incongruous. The config option seems heavy-handed here, though. 🤔

Reply:
Other suggestion. Maybe we can suggest config to disable `yarn`? And I will use it in `matrix` to disable it for old Node.js versions?

Reply:
I made brunch to show current problem with `yarn`: https://travis-ci.org/ai/nanoid/builds/441188572

Reply:
Right. So, dropping this line seems like a good first step towards cleaning up the logic. Would you like to have this PR reopened, or would you like to open a new one?

Reply:
I think reopening will be the best option to keep our good conversation :).

Do you want me to update PR and remove using this line to `install`?

Reply:
I was thinking just dropping this line.